### DISCUSSION TITLE
Remove tooltip in custom igx-icon

### DISCUSSION THREAD

**Discussion by ssg127** [IgniteUI/igniteui-angular#11862] [Q&A] [answered]
I am overwriting the column icon in the data grid. I'm not using one of the ignite ones, I'm using my own SVG. The problem is that when I put the cursor over the icon a tooltip appears and I don't want it to appear. In the following image you can see it:

![image](https://user-images.githubusercontent.com/36885861/179010137-2e45a8c1-6511-4ada-b9a0-d1b441964ef1.png)

To load the icon I do this:
`this.iconService.addSvgIcon('filter', '/assets/icons/filter.svg');`

And to use it this:
`<igx-icon name="filter"></igx-icon>`

How can I remove that tooltip and not show it as in the rest of the icons?

I'm using -> "@infragistics/igniteui-angular": "^12.3.27",

**Comment by zdrawku** [comment, marked as answer]
Go ahead and add the fourth parameter `stripMeta` when calling addSvgIcon. This will remote the `title` attribute from the svg element.

[Stackblitz example](https://stackblitz.com/edit/angular-1hbnqx?file=src/app/grid/grid-editing-sample/grid-editing-sample.component.ts)

![image](https://user-images.githubusercontent.com/11193764/180174039-6dde8c34-77df-4364-9505-9aaa4ad8099c.png)
